ngrinder-runtime/src/main/java/org/ngrinder/http/HTTPResponse.java

```diff
@@ -38,7 +41,7 @@
 		this.message = message;
 	}
 
-	public byte[] getBodyBytes() {
+	public byte[] getBody() {
```

Comment:
Why is this changed?

Reply:
I think the suffix `bytes` is unnecessary. When I suddenly watch this method name again, the word `bodyBytes` seems to be unnatural.

Reply:
Sure.. usually I think develops may expect getBody returns Body instance not bytes[]

Reply:
Got it. The way to make this more intuitive could be to add a Body. I'll reset it.